README.md

```diff
@@ -6,8 +6,6 @@ Utility library to improve the robustness of your Angular forms.
 
 Whether you have simple and tiny forms or huge and complex ones, `ngx-sub-form` will help you build a solid base for them.
 
-- 🗜️ Tiny bundle  
```

Comment:
isn't it still tiny?

Reply:
Strange, yea it’s no big deal but I would be surprised if the bundle got a bunch bigger

Reply:
yup it's just that we specified 15kb earlier, 30kb since we got the 2 APIs but now I found numbers like 150kb so I was like :shrug: what's going on. How did I get these numbers in the first place, unsure